dts/arm/nxp/nxp_imx6sx_m4.dtsi

```diff
@@ -241,6 +241,30 @@
 			#gpio-cells = <2>;
 			status = "disabled";
 		};
+
+		epit1:epit@420d0000 {
+			compatible = "nxp,imx-epit";
+			reg = <0x420d0000 0x4000>;
+			interrupts = <56 0>;
+			rdc = <(RDC_DOMAIN_PERM(A9_DOMAIN_ID,\
+					       RDC_DOMAIN_PERM_RW)|\
+			       RDC_DOMAIN_PERM(M4_DOMAIN_ID,\
+					       RDC_DOMAIN_PERM_RW))>;
+			label = "EPIT_1";
+			status = "disabled";
+		};
+
+		epit2:epit@420fffff {
```

Comment:
I think there's a typo in the UM, I'm guessing the address should be 420D_4000 for EPIT2, since it shows up at 020D_4000.

Reply:
You are right, thanks. Fixed.